### ISSUE TITLE
[MainUI] Add cancel button on login screen

### ISSUE THREAD

**Issue by EngiNearEngiFar** [openhab/openhab-core#5427]
## The problem
 There is no cancel button on the login page. 
![image](https://user-images.githubusercontent.com/83063002/137018936-ccc85b67-23f5-4c59-a531-62d5c8fda116.png)

When using the main UI with a tablet in kiosk mode, there are no control button, so you are "trapped" on the login screen. There is only the keypad input, but I don't see an option to go back to the overview page.

## Your suggestion

Adding an "cancel" button next to the "login" button to go back to main sceen.

**Comment by jsjames** [comment]
@florian-h05 - Correct me if I am wrong, but this page is served up from the server and not part of the webui app.  If so, we should as to move to core, correct?

**Comment by florian-h05** [comment]
Exactly, this page is served by core. org.openhab.core.auth.http or something like this

**Comment by florian-h05** [comment]
@kaikreuzer Please move this to core and assign me.
